src/main/java/seedu/duke/utility/InputParser.java

```diff
@@ -29,71 +29,66 @@ public boolean isByeTime() {
     }
 
 
-    public void parseInput(String input) {
+    public String parseInput(String input) {
 
-        String command = StringOperations.getFirstWord(input);
-        switch (command.toLowerCase()) {
+        String command = StringOperations.getFirstWord(input).toLowerCase();
+
+        switch (command) {
         case "bye":
             Ui.printByeMessage();
             this.isBye = true;
-            return;
+            return command;
```

Comment:
Should we use break instead of return?

Reply:
No, I think break might cause the program to terminate as if a "bye" has been passed in. Not sure if it will break the outerloop in the main Duke class.

Reply:
As long as it works I guess ...